app/form_objects/result_filters/location_filter_form.rb

```diff
@@ -63,5 +64,13 @@ def location_query
     def search_radius
       @params[:rad]
     end
+
+    def country(results)
```

Comment:
I'm wondering there's a decent place create a `DEVOLVED_NATIONS` constant so we can reuse it here
```
flattened_results = results.address_components.map(&:values).flatten
NameofModel::DEVOLVED_NATIONS.each { |nation| return nation if flattened_results.incldue(nation)
```

Reply:
Good idea, I've put it in the initialisers, similar to GCSE grades on Apply.